Neos.Neos/Classes/Service/LinkingService.php

```diff
@@ -280,19 +268,9 @@ public function createNodeUri(
         $format = null,
         $absolute = false,
         array $arguments = [],
-        $section = '',
-        $addQueryString = false,
-        array $argumentsToBeExcludedFromQueryString = [],
```

Comment:
if we feel relly nice, we could keep these options in the already deprecated linking service. Wdyt?